Gemfile

```diff
@@ -10,7 +10,7 @@ def vanagon_location_for(place)
   end
 end
 
-gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || '0.11.3')
+gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || '~> 0.12.1')
```

Comment:
According to this https://www.devalot.com/articles/2012/04/gem-versions.html the last-specified number is allowed to vary.